packages/modules/itwin-viewer-react/src/components/Viewer.tsx

```diff
@@ -85,6 +87,7 @@ export const Viewer = ({
           name: extension.name,
           loaded: false,
           version: extension.version,
+          args: extension.args,
```

Comment:
Do we ever expect the args to change? I guess right now you've set it up for startup extensions where the args would always be constant. I'm probably getting way too ahead of myself when thinking about dynamically loaded extensions, but in that scenario can we expect args to be different every time they're loaded?

Reply:
The args can be dynamic but if someone wants to call/load the Extension another time they can directly use `IModelApp.extensionAdmin` to make the call.

Reply:
Ideally we support changing the extension prop and add/removing the extensions. That is not the case at the moment. I'm not sure there is a way to "unload" an extension in the API at the moment? @calebmshafer ?

Reply:
Yes, currently there is not a way to unload or remove an Extension from the app. Not quite sure I understand the use-case, we can follow up separately about it if you'd like.